api/scpca_portal/management/commands/load_data.py

```diff
@@ -97,6 +97,7 @@ def download_data(bucket_name, scpca_project_id=None, scpca_sample_id=None):
                 (
                     "--exclude=*",  # Must precede include patterns.
                     "--include=project_metadata.csv",
+                    f"--include=merged/{scpca_project_id}*",
```

Comment:
Doesn't merged data only apply at the project level? Here it's downloading the project when triggered from the sample level.

Reply:
Yeah good catch, I think that we can leave this for now. Issue that I have is that when recreating a sample we would need to update the project anyway. So honestly, at this time we never use this flag and I'd like to remove it in it's entirety anyway. I haven't even tested processing of a single sample because of the reason just stated.

Reply:
Right, I've been wondering the same thing about the command to download individual samples for that exact reason. Sounds good.